mk/demo.mk

```diff
@@ -10,7 +10,7 @@ rbd: CMD = bash -c "rbd -p $(RBD_POOL) info $(RBD_IMAGE_NAME) || rbd -p $(RBD_PO
 demo: export NVMEOF_HOSTNAME != docker ps -q -f name=ceph-nvmeof_nvmeof_1
 demo: rbd ## Expose RBD_IMAGE_NAME as NVMe-oF target
 	$(NVMEOF_CLI) create_bdev --pool $(RBD_POOL) --image $(RBD_IMAGE_NAME) --bdev $(BDEV_NAME)
-	$(NVMEOF_CLI) create_subsystem --subnqn $(NQN)
+	$(NVMEOF_CLI) create_subsystem --subnqn $(NQN) --serial $(SERIAL)
```

Comment:
We removed the --serial parameter on purpose. It is now optional and if it's not there we generate a random one.